.circleci/config.yml

```diff
@@ -99,13 +103,33 @@ jobs:
           command: "prove -v . |& tee ../nginx-plus-module-prove-test-verbose || true"
           environment:
             TEST_NGINX_BINARY: "/usr/sbin/nginx"
-            TEST_NGINX_GLOBALS: "load_module /tmp/workspace/ubuntu2204/<<parameters.version>>/ngx_http_akita_module.so;"
+            TEST_NGINX_GLOBALS: "load_module /tmp/workspace/ubuntu_2204/<<parameters.version>>/ngx_http_akita_module.so;"
             TEST_NGINX_GLOBALS_HTTP: "akita_agent localhost:5555; akita_enable off;"
           working_directory: "nginx-tests-tip"
       - store_artifacts:
           path: nginx-plus-module-prove-test-verbose
           destination: "nginx-test-results-<<parameters.version>>/nginx-plus-module-prove-test-verbose"
 
+  release-module:
+    docker:
+      - image: cimg/go:1.20
+    resource_class: medium
+    steps:
+      - checkout:
+          path: "akita"      
+      - attach_workspace:
+          at: ./artifacts
+      - run:
+          name: "Build GHR"
+          command: go install github.com/tcnksm/ghr@latest
+      - run:
+          # Fails if release already exists; bump VERSION file on the release branch in order to kick off a new release.
+          name: "Create release"
+          command: |
+            VERSION=`cat akita/VERSION`
+            echo "Releasing ${VERSION}"
+            ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} ${VERSION} ./artifacts/
```

Comment:
it's probably safer to quote all these vars to avoid potential weirdness if variables end up empty or trigger bash word splitting rules, eg

```
ghr -t "${GITHUB_TOKEN}" -u "${CIRCLE_PROJECT_USERNAME}" -r "${CIRCLE_PROJECT_REPONAME}" -c "${CIRCLE_SHA1}" "${VERSION}" ./artifacts/
~``